lbz/dev/misc.py

```diff
@@ -40,3 +40,12 @@ def __init__(
 
     def __repr__(self) -> str:
         return f"<Fake Event {self['method']} @ {self['path']} body: {self['body']}>"
+
+    def __eq__(self, other: object) -> bool:
+        if isinstance(other, dict):
+            self_copy = self.copy()
+            self_copy.pop("requestContext")
+            other_copy = other.copy()
+            other_copy.pop("requestContext")
+            return hash(json.dumps(self_copy)) == hash(json.dumps(other_copy))
```

Comment:
That doesn't seem to be the right way to compare dictionaries, especially that simple `dict` does not guarantee the order of elements:
```python
dict_1 = {"a": 1, "b": 2}
dict_2 = {"b": 2, "a": 1}

assert dict_1 == dict_2  # True
assert hash(json.dumps(dict_1)) == hash(json.dumps(dict_2))  # False
```